src/workers/postUpdated.ts

```diff
@@ -479,6 +488,15 @@ const updatePost = async ({
     data.contentQuality.manual_clickbait_probability =
       databasePost.contentQuality.manual_clickbait_probability;
   }
+  if (smartTitle) {
+    if (Object.keys(databasePost.translation).length !== 0) {
+      data.translation = databasePost.translation;
+      data.translation.en = data.translation.en || {};
+      data.translation.en.smartTitle = smartTitle;
```

Comment:
non blocking but can't we just spread this, feels like it would be easier?

it would also remove an option of needing if-else

Reply:
Think it'd look something like this. Haven't tested or validated that it works.

```js
data.translation = {
  ...databasePost.translation,
  en: {
    ...databasePost.translation.en,
    smartTitle,
  },
};
```

Reply:
yep

Reply:
👌